pagopa/io-app
# chore(Federated Identity Management System): [IAFIMS-4] Custom internal link for sso through FIMS in IO

## Short description
This PR aims to introduce a new dedicated url protocol to navigate to FIMS webview screen.

## List of changes proposed in this pull request
- Enables the new `iosso://` url protocol for internal purpose.

## How to test
- Enables FIMS local FF
- execute from anywhere:
```typescript
handleInternalLink(
          dispatch,
          `iosso://http://127.0.0.1:3000/myportal_playground.html`
        );
```
check the FIMS webview screen is correctly displayed.

This PR depends on #3850
